streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/entity/Variable.java

```diff
@@ -42,7 +42,6 @@ public class Variable implements Serializable {
     private String variableCode;
 
     @NotBlank(message = "{required}")
-    @Size(max = 50, message = "{noMoreThan}")
```

Comment:
Why delete this code?

Reply:
I added the real variables of the production system and found that 50 characters are really not enough. For example, a kafka cluster has 20 ips